src/main/java/practice/StreamPractice.java

```diff
@@ -1,29 +1,60 @@
 package practice;
 
-import java.util.Collections;
+import java.util.Arrays;
 import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.OptionalDouble;
+import java.util.OptionalInt;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
 import model.Candidate;
+import model.Cat;
 import model.Person;
 
 public class StreamPractice {
+
     /**
      * Given list of strings where each element contains 1+ numbers:
      * input = {"5,30,100", "0,22,7", ...}
      * return min integer value. One more thing - we're interested in even numbers.
      * If there is no needed data throw RuntimeException with message
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
-    public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+    public static int findMinEvenNumber(List<String> numbers) {
+        OptionalInt min = numbers.stream()
+                .map(s -> s.split(","))
+                .flatMap(Arrays::stream)
```

Comment:
```suggestion
                .flatMap(s -> s.split(",").stream())
```
use just 1 map or flatMap if possible